src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -371,6 +372,23 @@ public OffenderAssessments getAssessmentsByCrn(
             .orElseThrow(() -> new NotFoundException(String.format("Offender with crn %s not found", crn)));
     }
 
+    @ApiOperation(value = "Update offender tier", tags = {"Assessments"})
+    @ApiResponses(
+        value = {
+            @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
+            @ApiResponse(code = 404, message = "The offender CRN is not found", response = ErrorResponse.class),
+            @ApiResponse(code = 500, message = "Unrecoverable error whilst processing request.", response = ErrorResponse.class)
+        })
+    @PostMapping(path = "/offenders/crn/{crn}/tier/{tier}")
+    @PreAuthorize("hasRole('ROLE_MANAGEMENT_TIER_UPDATE')")
+    public OffenderDetail updateTier(
+        @ApiParam(value = "CRN for the offender", example = "A123456", required = true)
+        @NotNull @PathVariable(value = "crn") final String crn,
+        @ApiParam(value = "New tier", example = "A1", required = true)
```

Comment:
maybe document the list of tier codes (even though it is reference data it does mean the docs offer a quick glance at the possible values)

Reply:
Done